GMC_identification.ipynb

```diff
@@ -0,0 +1,1724 @@
+       " n_y:    144  type_y: DEC--SIN  unit_y: deg    range:    69.698621 deg:   69.726427 deg\n",
+       " n_s:     69  type_s: VRAD      unit_s: m / s  range:   101453.231 m / s:  397912.263 m / s"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from spectral_cube import SpectralCube\n",
+    "\n",
+    "#####\n",
+    "# Final version of this tutorial will access this data set from the DataVerse\n",
+    "#####\n",
+    "# from astropy.utils.data import download_file\n",
+    "\n",
+    "# datafile = download_file(\n",
+    "#     'https://dataverse.harvard.edu/api/access/datafile/10359120',\n",
+    "#     cache=True, show_progress=True)\n",
+    "\n",
+    "datafile = \"temp_data/DFE1_com_co21_5kms_natural_pbcorr.image.fits\"\n",
+    "\n",
+    "cube = SpectralCube.read(datafile)\n",
+    "\n",
+    "cube"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9d74f40d-b5d8-482e-98d8-7c70be603fc7",
+   "metadata": {},
+   "source": [
+    "To simplify the GMC identification and analysis, we need to apply 2 preprocessing steps to this data cube.\n",
+    "\n",
+    "1. Convolve to a common, circular beam (i.e., PSF): From the summary above, a `VaryingResolutionSpectralCube` means that the beam of the data varies across frequency/velocity channels. We will convolve the cube to have a common beam across channels, and circularize the beam\n",
+    " \n",
+    "2. Convert from flux density to brightness temperature: The cube currently has units of `Jy/beam`, or a specific flux density. It is convenient to instead use brightness temperature units (using the Rayleigh-Jeans approximation) for resolved structure."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "67d46e59-f173-437e-8a30-81f8d2b49877",
+   "metadata": {},
+   "source": [
+    "### Convolve to a circular, common beam size\n",
+    "\n",
+    "The largest beam size in the cube is:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "5f34b568-52c7-4f71-8f95-34dc66650c6b",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/latex": [
+       "Beam: BMAJ=$4.380295753479004^{''}$ BMIN=$2.885484218597412^{''}$ BPA=$-48.6453857421875^\\circ$"
+      ],
+      "text/plain": [
+       "Beam: BMAJ=4.380295753479004 arcsec BMIN=2.885484218597412 arcsec BPA=-48.6453857421875 deg"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "cube.beams.largest_beam()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a8478f88-8048-45d4-9610-2a8af0a172c3",
+   "metadata": {},
+   "source": [
+    "In this case, we will define the target resolution to be a beam with a FWHM of $4.^{''}5$, slightly larger than the current largest beam. Since we want a circular beam, we do need to specify the minor axis or position angle."
```

Comment:
do **not** need to specify the minor axis or PA.

Reply:
fixed!